src/components/ui/calendar/calendar.component.tsx

```diff
@@ -67,6 +67,14 @@ export type CalendarElement<D = Date> = React.ReactElement<CalendarProps<D>>;
  * @property {(D, NamedStyles) => ReactElement} renderYear - Function component
  * to render instead of default year cell.
  * Called with a date for this cell and styles provided by Eva.
+ * 
+ * @property {(CalendarViewMode) => void} renderArrowLeft - Function component
+ * to render instead of default left arrow.
+ * viewMode returns string with current calendar view ("YEAR", "MONTH", "DATE").
```

Comment:
Confusing docs. I'd say we better go with
"Accepts viewMode argument which is..."

Reply:
I would replace `left arrow` with something  more contextual like `previous`